pkg/playground/speed.js

```diff
@@ -200,7 +200,7 @@ function download(ev) {
 function spawn() {
     stop();
 
-    document.getElementById("spawn-result").innerHTML = "Running...";
+    document.getElementById("spawn-result").textContent = "Running...";
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19047-20230630-121653-41e6f122-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)